lnwire/onion_error.go

```diff
@@ -15,6 +15,7 @@ import (
 // failure code.
 type FailureMessage interface {
 	Code() FailCode
+	Error() string
```

Comment:
It isn't necessary to add this method to the interface type itself. Instead, just adding the `Error` method directly to all the structs suffices.